src/Engine/ProtoCore/FFI/CLRObjectMarshaler.cs

```diff
@@ -1332,7 +1332,7 @@ void core_Dispose(ProtoCore.RuntimeCore sender)
         {
             CLRObjectMarshaler marshaller = null;
             if (!mObjectMarshlers.TryGetValue(sender, out marshaller))
-                throw new KeyNotFoundException();
+                throw new System.Collections.Generic.KeyNotFoundException();
```

Comment:
hmm This file has `System.Collections.Generic` using already I believe

Reply:
this is to be explicit because we also have a key not found exception.

Reply:
@mjkkirschner Then should we rename our own exception in a different task?

Reply:
please a different task. our is 
`DesignScript.Builtin.KeyNotFoundException` 
@aparajit-pratap is aliasing them above.